sdk/go/asymmetric_decrypt.go

```diff
@@ -0,0 +1,64 @@
+package ehsm
+
+import (
+	"fmt"
+
+	"github.com/iancoleman/orderedmap"
+)
+
+/*
+Description:
+Decrypt an arbitrary set of bytes using the CMK.(only support asymmetric types).
+Input:
+keyid    		-- The keyid of the asymmetric cmk.
+
+ciphertext 		-- The data of the ciphertext in BASE64 string.
+
+padding mode	-- Padding_mode is necessary when keyspec is RSA.
```

Comment:
add comments to make it clear for sm2 or ecc key user no need to input the padding mode